zeebe/exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/TestSupport.java

```diff
@@ -132,7 +132,9 @@ static Stream<ValueType> provideValueTypes() {
             ValueType.SBE_UNKNOWN,
             ValueType.NULL_VAL,
             ValueType.PROCESS_INSTANCE_RESULT,
-            ValueType.CLOCK);
+            ValueType.CLOCK,
+            // these are not yet supported
+            ValueType.ROLE);
```

Comment:
Adding this here as I'm not sure if following the old approach is still valid

Reply:
I informed about this [here](https://camunda.slack.com/archives/C05QBRGMA9F/p1728383396104949?thread_ts=1728374201.862919&cid=C05QBRGMA9F). We don't need to support these records in the old exporters as long as they don't get consumed. The only component still using the old exporters is Optimize and since they won't consume these records we can safely ignore them and don't need to support them in the ES/OS exporters.